### PR TITLE
Issues/1740 live tailing improvements

### PR DESCRIPTION
**Pull request by Mgrdich**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Generalize instead of prop drill i've made a context to share the SeekDirection , and made minor code modifications.
and Fixed the issue.

**Is there anything you'd like reviewers to focus on?**
if the old functionality still exists , i did check it myself everything works fine. but a second eye wouldn't hurt @Haarolean 

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [x] Unit checks
- [ ] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by Mgrdich** [comment]
The Test Coverage can be done by another issue since this file in it self had a lot of things uncovered.

File: kafka-ui-react-app/src/components/contexts/TopicMessagesContext.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { SeekDirection } from 'generated-sources';
+
+export interface ContextProps {
+  seekDirection: SeekDirection;
+  searchParams: URLSearchParams;
+  toggleSeekDirection(val: string): void;
```
Comment (workshur) [inline kafka-ui-react-app/src/components/contexts/TopicMessagesContext.ts:7]: `toggle` usually implies only two possible states **on** || **off** (true || false);

Let's try to fine more suitable name for this method. Eg: `setSeekDirection`

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/MessagesTable.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    it('should check the render', () => {
+      expect(screen.getByRole('table')).toBeInTheDocument();
+    });
+
+    it('should check the if no elements is rendered in the table', () => {
+      expect(screen.getByText(/No messages found/i)).toBeInTheDocument();
+    });
+  });
+
+  describe('Custom Search Params', () => {});
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/MessagesTable.spec.tsx]: Pls describe)

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/Messages.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    });
+    it('should check default seekDirection if they actually take the value from the url', () => {
+      expect(screen.getByRole('listbox')).toHaveTextContent(
+        SeekDirectionOptionsObj[SeekDirection.FORWARD].label
+      );
+    });
+  });
+
+  describe('Custom search Params', () => {
+    it('should check custom seekDirection if they actually take the value from the url', () => {
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/Messages.spec.tsx:68]: from my point of view, it is better when the name of test case describes the behavior of the component, and not the actions inside the test case

**Review by workshur — APPROVED** [review]
Looks good in general

File: kafka-ui-react-app/src/components/contexts/TopicMessagesContext.ts (hunk 1 above)
Comment (Mgrdich) [reply kafka-ui-react-app/src/components/contexts/TopicMessagesContext.ts:7]: okay will do.

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/Messages.spec.tsx (hunk 3 above)
Comment (Mgrdich) [reply kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/Messages.spec.tsx:68]: yeah it makes total sense , i will fix it.

**Review by sasunprov — APPROVED** [review]

**Pull request merged**